### ISSUE TITLE
Formatting Problem with multiline Selectors in media Querys 

### ISSUE THREAD

**Issue by Kingju777** [glenn2223/vscode-live-sass-compiler#106]
If you put a Selector with more than one line in a ‘@media’ Query the space that is needed to mark the indentation is not put there.

CSS Result (formatting in front of the `body` element wrong)
```
@media screen {
  html,
body {
    height: 100%;
    width: 100%;
  }
```

Corrected Result (corrected by me)
```
@media screen {
  html,
  body {
    height: 100%;
    width: 100%;
  }
```

**Comment by glenn2223** [comment]
Oh yeah, I can verify that issue. the compilation itself is handled by `sass/dart-sass` so I would recommend you open this issue [here](https://github.com/sass/dart-sass/issues).

When you open the issue, reference this issue and also mention that it's the `Javascript API` where your issue lies

**Comment by Kingju777** [comment]
@glenn2223 this issue was fixed in 1.56.0 (1.58.0 is the current Version) in sass/dart-sass, can you please update your dependencies.

Thank you!
